app/assets/javascripts/components/LayerSwitcher.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+class LayerSwitcher extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    let switchers = [];
+
+    this.props.layersList.map( (layer, i) => {
+      const slug = layer.slug
+
+      switchers.push ( <div className="switcher" key={i}>
+        <input type="checkbox" id={slug}  checked={ layer.active } onChange={ () => {this.props.toggleLayers( {slug} )} }/>
```

Comment:
I thought the same, but it is not "active/inactive" background but that we have two different categories. "To fix/ To manage" with two different colors. 
So as I'm adding the layers from the API right now, I tough I could do it later.